src/js/components/Pagination/stories/typescript/Custom.tsx

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
 
-import { Box, Grommet, Pagination, Text } from 'grommet';
+import { Box, Grommet, Pagination, Text, ThemeContext } from 'grommet';
 import { grommet } from 'grommet/themes';
 import { deepMerge } from 'grommet/utils';
 import { FormPreviousLink } from 'grommet-icons/icons/FormPreviousLink';
 import { FormNextLink } from 'grommet-icons/icons/FormNextLink';
-import { ThemeContext } from 'styled-components';
 
 const customTheme = deepMerge(grommet, {
```

Comment:
You should add 'ThemeType',  it will surface any potential issues with theme typing.
See how it is done on other TS examples.

Reply:
Got it, I've just tested locally and fixed one item that was flagged. However, it only flags when we're not using `deepMerge`. We'd deepMerged with grommet theme to inherit the grommet font as opposed to the browser default. Would you like me to push up the code with `ThemeType` and not use `deepMerge`? Or is it fine that I tested locally and we can leave the published story as is?

Reply:
Let's avoid using deepMerge and grommet theme, you can explicitly add the font to the theme.

Reply:
Update, I will just add "font-family: "Arial"" to the TS theme to make it look a little nicer and allow us to leave ThemeType.

Reply:
Please add a comment to the ThemeType usage:
```
// Type annotations can only be used in TypeScript files.
// Remove ': ThemeType' if you are not using Typescript.
```